app/assets/stylesheets/application.css

```diff
@@ -32,4 +32,8 @@ body .goog-te-banner-frame {
   text-decoration: underline;
   color: inherit;
   font-weight: 700;
+}
+
+.habitat-income-list li {
+  float: none;
```

Comment:
float:left seemed to be the culprit, but I didn't want to change the inline-list style without doing more research into what else might be affected, so I just added a class instead.